README.md

```diff
@@ -11,3 +11,9 @@ File 2 - .env.development
 inside the file add:
 
 PGDATABASE=nc_news
+
+////////////////////////////////////////
+
+functions:
```

Comment:
I like your thinking, but please don't worry about listing your functions in here. Later on in the sprint you'll work on a ticket (I think 13) that lists everything your api is capable of in the endpoints.js file.